### ISSUE TITLE
Currently selected item is not highlighted when accessing the application with Keyboard Only 

### ISSUE THREAD

**Issue by VaishaliSanthosh** [topcoder-platform/community-app#2554]
**Describe the bug**
Currently selected item is not highlighted when accessing the application with Keyboard Only 

**To Reproduce**
1. Go to 'https://test-community-app.topcoder-dev.com/challenges?'
2. After sign in  Navigate to last challenge in the page using keyboard only
3. Press tab key
4. Cannot see where the navigation is leading to, no menu/options/texts are highlighted 

**Expected behavior**
Current selection when navigating using keyboard only should be highlighted

**Screenshots**
Till Step 2:
![Screenshot 1](https://user-images.githubusercontent.com/29533254/59141946-52b1ce80-89d3-11e9-82bf-a0b3b83ec6fb.png)
After Step 3:
![Screenshot 2](https://user-images.githubusercontent.com/29533254/59141950-665d3500-89d3-11e9-905d-9c23e06a81b4.png)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Chrome
 - Version :75.0.3770.80

**Comment by nithyaasworld** [comment]
Out of Scope
